File: programmers 문제 모음/연습문제/2016년.py
from datetime import datetime

def solution(a, b):
    day = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]
    before = datetime(2016, 1, 1)
    after = datetime(2016, a, b)
    diff = (after - before).days 
    answer = day[diff % 7]
    return answer


# a = 5
# b = 24
# print(solution(a, b))

File: programmers/연습문제/2016년.js
function solution(a, b) {
    let day = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"]
    let date2 = new Date(`2016,${a},${b}`);
    return day[date2.getDay() % 7];
}


// let a = 5
// let b = 24
// console.log(solution(a, b))